pkg/controllers/namespace/namespace_controller.go

```diff
@@ -132,6 +132,33 @@ func (r *NamespaceReconciler) createDefaultSubnetSet(ns string, defaultSubnetSiz
 	return nil
 }
 
+func (r *NamespaceReconciler) deleteDefaultSubnetSet(ns string) error {
+	subnetSets := []string{
+		types.DefaultVMSubnetSet,
+		types.DefaultPodSubnetSet,
+	}
+	for _, name := range subnetSets {
+		if err := retry.OnError(retry.DefaultRetry, func(err error) bool {
+			return err != nil
+		}, func() error {
+			obj := &v1alpha1.SubnetSet{}
+			err := r.Client.Get(context.Background(), client.ObjectKey{
+				Namespace: ns,
+				Name:      name,
+			}, obj)
+			if err != nil {
+				return client.IgnoreNotFound(err)
+			}
+			log.Info("delete default SubnetSet", "Namespace", ns, "Name", name)
+			return r.Client.Delete(context.Background(), obj)
+		}); err != nil {
```

Comment:
do you need to delete it from the store?

Reply:
It initiates a delete event, but the actual deletion occurs during the subnet set reconciliation process.